docs/guides/deploy/How-to-install-Erda-zh.md

```diff
@@ -113,13 +113,13 @@
 
    ```shell
    # 安装 erda-base
-   helm install package/erda-base-0.1.0.tgz --generate-name
+   helm install package/erda-base-$(cat VERSION).tgz --generate-name
    
    # 安装 erda-addons
-   helm install package/erda-addons-0.1.0.tgz --generate-name
+   helm install package/erda-addons-$(cat VERSION).tgz --generate-name
    
    # 安装 erda
-   helm install package/erda-0.1.0.tgz --generate-name
+   helm install package/erda-$(cat VERSION).tgz --generate-name
    ```
```

Comment:
have you mentioned where this script executed from?

Reply:
Doc above already comment out. And `VERSION` file is in the same folder as `package` dir. So no new broken things.